featuregates/controller/pkg/feature/doc.go

```diff
@@ -0,0 +1,5 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package feature has the controller for Feature in core API group.
```

Comment:
**nitpick**: Maybe explain that this is responsible for determining the feature's activation state based on its stability level and any feature gates?